packages/@aws-cdk/cloudformation-include/test/test-templates/bucket-with-parameters.json

```diff
@@ -0,0 +1,19 @@
+{
+  "Parameters": {
+    "BucketName": {
+      "Default": "MyS3Bucket",
+      "Description": "The name of your bucket",
+      "Type": "String"
```

Comment:
Also use the other properties of Parameters here (`ConstraintDescription`, `NoEcho`, etc.).